CONTRIBUTING.md

```diff
@@ -12,25 +12,31 @@ issue trackers, chatrooms, and mailing lists.
 
 2. Run `./bin/setup`.
 
-3. Run the tests. We only take pull requests with passing tests, and it's great
-   to know that you have a clean slate:
-   `bundle exec rake && bundle exec appraisal rake`
+3. Run `appraisal install`
 
-4. Add a test for your change. Only refactoring and documentation changes
+4. Run the tests. We only take pull requests with passing tests, and it's great
+   to know that you have a clean slate: `bundle exec rake && bundle exec appraisal rake`
+
+5. Add a test for your change. Only refactoring and documentation changes
    require no new tests. If you are adding functionality or fixing a bug,
    we need a test!
 
-5. Make the test pass.
+6. Make the test pass.
```

Comment:
While we’re in here, thoughts on swapping all of these to `1`s and having Markdown build the ordered list for us?